src/config/llm.ts

```diff
@@ -101,6 +101,10 @@ export const getLLMConfig = () => {
       ENABLED_TAICHU: z.boolean(),
       TAICHU_API_KEY: z.string().optional(),
 
+      ENABLED_CLOUDFLARE: z.boolean(),
+      CLOUDFLARE_API_KEY: z.string().optional(),
+      CLOUDFLARE_BASE_URL_OR_ACCOUNT_ID: z.string().optional(),
```

Comment:
As cloudflare have lots models, it may be a good idea to add cloudflare_model_list env here.

Reply:
yes, like `OPENAI_MODEL_LIST`, refs to this PR: https://github.com/lobehub/lobe-chat/pull/3938